hadoop-ozone/common/pom.xml

```diff
@@ -135,6 +131,12 @@
       <artifactId>grpc-stub</artifactId>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <!-- required by io.netty.handler.ssl.SslContextBuilder in GrpcOmTransport -->
```

Comment:
If this is not directly used, shouldn't we have this as transitive dependency? 
We won't need this at runtime?

Reply:
Thanks @nandakumar131 for catching this, updated.